cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -728,7 +728,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         },
 
         hasChanges: function() {
-            return this.isModelLocked() !== this.isLocked() || this.getUnitLevel() !== this.currentUnitLevel();
+            return this.isModelLocked() !== this.isLocked() || this.getUnitLevel() !== this.currentUnitLevel() || this.getLogo() !== this.currentLessonLogo();
```

Comment:
it's redundant if I'm not mistaken because we use another settings location for lesson_logo settings, check it please and remove if so.

Reply:
done